script.js

```diff
@@ -0,0 +1,79 @@
+/* add todo*/
+const todoInput=document.getElementById("todo-input");
+const addTodoButton=document.getElementById("todo-button");
+const doingTodoList=document.getElementById("doing");
+const doneTodoList=document.getElementById("done");
+let todoArray=[];
+const doingCount=document.getElementById("doing-count");
+const doneCount=document.getElementById("done-count");
+
+/*create todo li*/
+function createTodoList(todoObj){
+    const todoListItem=document.createElement("li");
+    const deleteImg=document.createElement("img");
+    todoListItem.innerText="🐈 "+todoObj.text;
+    todoListItem.id=todoObj.id;
+    todoListItem.addEventListener("click", toggleTodo);
+    deleteImg.src="./img/bin.png";
+    deleteImg.addEventListener("click", deleteTodo);
+    todoListItem.appendChild(deleteImg);
+    todoObj.type? doingTodoList.appendChild(todoListItem):doneTodoList.appendChild(todoListItem);
+}
+
+/*click add todo button*/
+function onClickAddTodo(e){
+    e.preventDefault();
+    if (todoInput.value){
+        const todoObj={
+        "text":todoInput.value,
+        "id":Date.now(),
+        "type": true,
```

Comment:
```suggestion
        text:todoInput.value,
        id:Date.now(),
        type: true,
```

object 속성은 따옴표를 따로 붙이지 않아도 됩니다!!

Reply:
앗..ㅎㅎ 습관적으로 따옴표를 붙여버렸네요 이부분도 수정하겠습니다~.~